Code/Engine/Audio/AudioMetadataUtils.cpp

```diff
@@ -101,6 +102,34 @@ Texture* GetImageFromFileMetadata(const std::string& fileName)
                 }
             }
         }
+        else if (audioFile.hasID3v2Tag())
+        {
+            static const char* IdPicture = "APIC";
+            TagLib::ID3v2::Tag* id3v2tag = audioFile.ID3v2Tag();
+            TagLib::ID3v2::FrameList Frame;
+            TagLib::ID3v2::AttachedPictureFrame* PicFrame;
+
+            // picture frame
+            Frame = id3v2tag->frameListMap()[IdPicture];
+            if (!Frame.isEmpty())
+            {
+                for (TagLib::ID3v2::FrameList::ConstIterator it = Frame.begin(); it != Frame.end(); ++it)
+                {
+                    PicFrame = (TagLib::ID3v2::AttachedPictureFrame *)(*it);
```

Comment:
I'd keep the pointer attached to the classname in this instance:
 (TagLib::ID3v2::AttachedPictureFrame*)(*it)